src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14InstanceofWithPatternMatching.java

```diff
@@ -1,6 +1,7 @@
 //non-compiled with javac: Compilable with Java14
 package com.puppycrawl.tools.checkstyle.grammar.java14;
 
+import java.util.ArrayList;
```

Comment:
please add Inputs with
`?` operator , usage in ELSE block,  while, switch, ...
see all at
https://docs.oracle.com/en/java/javase/14/docs/specs/patterns-instanceof-jls.html#jls-6.3.1

Reply:
> switch

After reviewing [this](https://docs.oracle.com/en/java/javase/14/docs/specs/patterns-instanceof-jls.html#jls-6.3.1), and trying the example from [here](https://openjdk.java.net/jeps/8213076):
```
String formatted;
switch (obj) {
    case Integer i: formatted = String.format("int %d", i); break;
    case Byte b:    formatted = String.format("byte %d", b); break;
    case Long l:    formatted = String.format("long %d", l); break;
    case Double d:  formatted = String.format("double %f", d); break;
    case String s:  formatted = String.format("String %s", s); break
    default:        formatted = obj.toString();
}
```
This will not compile:
```
➜  src /usr/lib/jvm/java-14-openjdk/bin/javac --enable-preview --source 14 InputJava14InstanceofWithPatternMatching.java
InputJava14InstanceofWithPatternMatching.java:208: error: : or -> expected
            case Integer i: formatted = String.format("int %d", i); break;
                        ^
InputJava14InstanceofWithPatternMatching.java:209: error: : or -> expected
            case Byte b:    formatted = String.format("byte %d", b); break;
                     ^
InputJava14InstanceofWithPatternMatching.java:210: error: : or -> expected
            case Long l:    formatted = String.format("long %d", l); break;
                     ^
InputJava14InstanceofWithPatternMatching.java:211: error: : or -> expected
            case Double d:  formatted = String.format("double %f", d); break;
                       ^
InputJava14InstanceofWithPatternMatching.java:212: error: : or -> expected
            case String s:  formatted = String.format("String %s", s); break
                       ^
InputJava14InstanceofWithPatternMatching.java:212: error: ';' expected
            case String s:  formatted = String.format("String %s", s); break
                                                                            ^
Note: InputJava14InstanceofWithPatternMatching.java uses preview language features.
Note: Recompile with -Xlint:preview for details.
6 errors
```
From reading [this](https://bugs.openjdk.java.net/browse/JDK-8213076
), it would seem that pattern matching for switch hasn't been enabled yet?

> please add Inputs with `?` operator , usage in ELSE block, while

Added.